tests/g_force.test.cpp

```diff
@@ -0,0 +1,54 @@
+#include <libhal/units.hpp>
+#include <boost/ut.hpp>
+
+namespace hal {
+
+void g_force_test() {
+    using namespace hal::literals;
+    using namespace boost::ut;
+    "g_force_type UDL test"_test = []() {
+        // Set Up
+        g_force test_g_force_value = 2.0_g;
+        float control_value = 2.0;
+        // Verify
+        expect(test_g_force_value == control_value);
```

Comment:
```suggestion
        expect(that % test_g_force_value == control_value);
```

The `that` object wraps the type next to it and outputs a message like `1.5 != 1.0` if it fails. Its helpful to diagnose errors.